index.html

```diff
@@ -968,17 +951,30 @@ <h4><code>Timing-Allow-Origin</code> Response Header</h4>
 whether a resource's timing information can be shared with the
 <a>current document</a>, is as follows:</p>
 <ol>
-<li>
-<p>If the resource is same origin, return <code>pass</code>.</p>
-</li>
-<li>
-<p>If the <a>Timing-Allow-Origin</a> header value list contains a
-case-sensitive match for the value of the <code><a data-cite=
-"RFC6454#section-4" data-lt="http-origin">origin</a></code> of the
-<a>current document</a>, or a wildcard ("<code>*</code>"), return
-<code>pass</code>.</p>
-</li>
-<li>Return <code>fail</code>.</li>
+    <li><p>Let <var>tainted</var> be false.</p></li>
+    <li><p>If the resource's <a>Request</a>'s <a
+    data-cite="FETCH#concept-request-window">window</a> is not an <a
+    data-cite="HTML/webapppapis.html#environment-settings-object">environment
+    settings object</a>, set tainted to true</p></li>
```

Comment:
@annevk - can you double check that this part makes sense? Added it to avoid trying to look at the origin of a client that's not a settings object

Reply:
I recommend using https://fetch.spec.whatwg.org/#concept-request-origin for comparisons.